finance/proposal-calls/cycle4/orgmycology.md

```diff
@@ -0,0 +1,122 @@
+
+Organizational Mycology is a consulting organization that works with many open science and open source software communities. Our team members have been in leadership roles within these communities and have helped other organizations build community management, engagement, and governance capacity. We do so with a combination of social science-based introspection and data collection, leadership coaching, and development of sustainable participatory community strategies (i.e., ways for organizations to continue building and evaluating their communities long after we fulfill a contract). These activities are beneficial both for project leaders at a strategic level (e.g., in developing new processes and making administrative decisions) and for community managers at a practical level (e.g., in collecting feedback from the community and fostering healthy community dynamics).
+
+### Project / Work
+
+*Building on our six-months of successful collaboration with the AstroPy team managing the community hire, we’ve developed a follow-on proposal*
+
+- **Continued Community Manager Support and Coaching** - The OrgMycology team members are available to the community lead on topics including systems automation, community wrangling, qualitative survey techniques and methods and other topics as they arise.
+- **Qualitative Inquiry Coaching** – Supporting Beryl Kanali to continue to grow her skills in interviewing, data gathering and report-out as she has conversations with community members.
+- **Skills Map Development** – Addressing [our recommendation to better communicate user-contributotr pipelines](https://astropy-report.orgmycology.com/sections/consultant_recommendations#develop-and-visualize-user-to-contributor-pipelines). We propose to develop skills maps which can help to map roles of Astropy community members and the ways in which educational material and community progressions take place. We propose to develop two skills maps, one for Astropy Developers (focusing on on-ramps for new developers) and one for Astropy Users (starting with Astropy Learn lessons). These will be developed with community involvement and feedback and shared openly as a resource of the community to own and update as needed. [Examples of skills maps](https://orgmycology.com/skills-maps)
+- **Technical skills support** – Helping and supporting Community Lead as she develops more automations and tools that help her do the work of community manager for the Astropy Community. Project priorities will be set with Community Lead and monthly progress reports will be shared back with the CoCo and community. 
+- **1-2 - Special projects prioritized by CoCo**
+  - Contribution workshops - desing, support and test-out short workshops for PRs, contributing to documentation, learning about automated testing suites etc.
```

Comment:
Do you have a sense of how these these workshops would fit into the ones the Astropy community already organises? Or would these be completely separate?

For context, I ran a few Astropy workshops where I walked through a PR with students, and am wondering is this could be done regularly and with more concrete materials.

Reply:
Yes these would be working to actively support the community manager, we'd plan, develop lesson plans and run them the first time, and build scaffolding and structures for other short online community workshops to emulate.